### PR TITLE
Feature/lti class list

### PR DESCRIPTION
**Pull request by jlaney**

### CODE REVIEW

File: myuw/views/lti/photo_list.py [hunk 1]
```diff
@@ -19,17 +19,22 @@ def get_context_data(self, **kwargs):
...
+        # End
+
+        blti_data['authorized_sections'] = [s.sis_section_id for s in sections]
+        self.set_session(request, **blti_data)
+
         return {
             'lti_session_id': request.session.session_key,
             'lti_course_name': blti_data.get('context_label'),
-            'sections': sections,
+            'sections': sorted(sections, key=lambda section: section.name),
```
Comment (fanglinfang) [inline myuw/views/lti/photo_list.py]: Could you add test for this?

File: myuw/views/api/instructor_schedule.py [hunk 2]
```diff
@@ -648,5 +651,15 @@ def is_authorized_for_section(self, request, schedule):
...
-        # method_decorator validates LTI session with course admin role
-        pass
+        blti_data = BLTI().get_session(request)
+
+        # TODO: verify section is in the list of viewable sections
+        valid_sections = blti_data.get('authorized_sections')
+
+        # For independent study, set the actual instructor if needed
+        inst_regid = blti_data.get('independent_study_instructor_regid')
+        if inst_regid is not None and inst_regid != schedule.person.uwregid:
```
Comment (fanglinfang) [inline myuw/views/api/instructor_schedule.py]: would it be possible to add some test for this?

File: myuw/views/lti/photo_list.py [hunk 3]
```diff
@@ -19,17 +19,22 @@ def get_context_data(self, **kwargs):
...
         except DataFailureException as err:
             pass
 
+        # Temp code, current version just uses the course id..
+        (sws_course_id, sws_inst_regid) = sws_section_label(sis_course_id)
+        blti_data['independent_study_instructor_regid'] = sws_inst_regid
+        # End
+
+        blti_data['authorized_sections'] = [s.sis_section_id for s in sections]
+        self.set_session(request, **blti_data)
```
Comment (fanglinfang) [inline myuw/views/lti/photo_list.py]: Is it worth to add some test for this?

Comment (jlaney) [reply myuw/views/lti/photo_list.py]: Yes, done.

**Pull request closed without merging**